<?xml version="1.0"?>


<launch>
	<group ns="base">
		<node name="f9p" pkg="ublox" type="ublox_ros">
			<param name="serial_port" value="/dev/ttyACM0"/>
			<param name="base_type" value="moving"/>
			<param name="chain_level" value="0"/>
			<param name="rover_quantity" value="1"/>
			<param name="local_host" value="localhost"/>
			<param name="local_port" value="16140"/>
			<param name="rover_host" value="localhost"/>
			<param name="rover_port" value="16145"/>
		</node> 
	</group>

	<group ns="rover">
		<node name="f9p" pkg="ublox" type="ublox_ros">
			<param name="serial_port" value="/dev/ttyACM1"/>
			<param name="chain_level" value="1"/>
			<param name="rover_quantity" value="0"/>
			<param name="local_host" value="localhost"/>
			<param name="local_port" value="16145"/>
			<param name="base_host" value="localhost"/>
			<param name="base_port" value="16140"/>
		</node>
		<node pkg="rosbag" type="record" name="record" output="screen" args="--all -o ../data/basevel/basevel.bag "/> 
	</group>
</launch>
